README.md

```diff
@@ -711,6 +847,11 @@ Driver class must implement the following API:
     * space name
     * space options
 
+Driver class should implement the following API:
```

Comment:
Just adds it to the previous list of `Driver class must implement the following API:`, rather than creating a new one.

Reply:
Fixed